src/GitVersionCore/LibGitExtensions.cs

```diff
@@ -113,7 +113,7 @@ public static Commit FindCommitBranchWasBranchedFrom([NotNull] this Branch branc
                         mergeBaseCommit = findMergeBase,
                         branch = otherBranch
                     };
-                }).Where(b => b != null).OrderByDescending(b => b.mergeBaseCommit.Committer.When).ToList();
+                }).Where(b => b.mergeBaseCommit != null).OrderByDescending(b => b.mergeBaseCommit.Committer.When).ToList();
```

Comment:
Can `b` be `null` as the original code suggests? Then it would need to be

` Where(b => b != null && b.mergeBaseCommit != null) `